app/assets/stylesheets/components.scss

```diff
@@ -57,12 +57,39 @@ header.main {
   border-bottom: 2px solid black;
 }
 
-
 .results {
+  .toggle-button {
+    font-size: 24px;
+    span {
+      padding-right: 10px;
+    }
+  }
+
   h2 {
     background: rgb(169, 220, 228);
     padding: 10px 15px;
   }
+  h3 {
+    &.isOpen {
```

Comment:
@joris-rotteveel are we using `.isOpen`

Reply:
Good catch, we are not using this. I will remove this.